libraries/AP_HAL_ChibiOS/hwdef/CubeOrange-joey/hwdef-bl.dat

```diff
@@ -0,0 +1,16 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H743 bootloader
+
+include ../CubeOrange/hwdef-bl.dat
+
+undef USB_STRING_PRODUCT
+undef USB_STRING_MANUFACTURER
+undef APJ_BOARD_ID
+undef USB_PRODUCT
+
+# USB setup
+USB_PRODUCT 0x1056
+USB_STRING_MANUFACTURER "CubePilot"
+USB_STRING_PRODUCT "Joey"
```

Comment:
Shouldn't the Bootloader have `-BL` specified to distinguish between application and Bootloader?

Reply:
you're right, or better to not have USB_STRING_PRODUCT at all, and it will automatically be Joey-BL

Reply:
Wouldn't it be CubeOrange-joey-BL here, though? The board is different in this case.